webauthn/helpers/mldsa.py

```diff
@@ -0,0 +1,31 @@
+from .cose import COSEAlgorithmIdentifier
+
+class ML_DSAPublicKey:
```

Comment:
I'm getting a little nitpicky here, but can we rename this class? I just think it looks a little weird with the underscore 🤷‍♂️ 
```suggestion
class MLDSAPublicKey:
```